capa/rules/__init__.py

```diff
@@ -194,6 +196,59 @@ def __repr__(self):
         return str(self)
 
 
+# COM data source https://github.com/stevemk14ebr/COM-Code-Helper/tree/master
+VALID_COM_TYPES = {
+    "class": {"db_path": "assets/classes.json.gz", "prefix": "CLSID_"},
+    "interface": {"db_path": "assets/interfaces.json.gz", "prefix": "IID_"},
+}
+
+
+def translate_com_feature(com_name: str, com_type: str) -> ceng.Or:
+    if com_type not in VALID_COM_TYPES:
+        raise InvalidRule(f"Invalid COM type present {com_type}")
+
+    CD = Path(__file__).resolve().parent.parent.parent
+    com_db_path = CD / VALID_COM_TYPES[com_type]["db_path"]
+    if not com_db_path.exists():
+        logger.error("Using COM %s database '%s', but it doesn't exist", com_type, com_db_path)
+        raise IOError(f"COM database path '{com_db_path}' does not exist or cannot be accessed")
+
+    with gzip.open(com_db_path, "rb") as gzfile:
+        com_db: Dict[str, List[str]] = json.loads(gzfile.read().decode("utf-8"))
+        guid_strings: Optional[List[str]] = com_db.get(com_name)
```

Comment:
this reads, decompresses, and deserializes the database *every* time that a COM feature is encountered. we may want to load the DB once and then reuse that, to avoid this overhead.

this isn't critical to fix now, but should be fixed before many COM features are used in the rule set.